src/main/java/mekanism/common/block/prefab/BlockBase.java

```diff
@@ -134,7 +135,10 @@ protected VoxelShape getShape(@NotNull BlockState state, @NotNull BlockGetter wo
     protected ItemInteractionResult useItemOn(@NotNull ItemStack stack, @NotNull BlockState state, @NotNull Level world, @NotNull BlockPos pos, @NotNull Player player,
           @NotNull InteractionHand hand, @NotNull BlockHitResult hit) {
         if (player.isShiftKeyDown() && MekanismUtils.canUseAsWrench(stack)) {
-            //Note: We don't handle checking if it is radioactive here, as the assumption is it doesn't have a tile so won't have that information
+            var tile = WorldUtils.getTileEntity(world, pos);
```

Comment:
As the comment being removed is correct still generally correct, I would prefer if we instead moved this logic into an override inside of `BlockTransmitter`. Also please don't use `var`